Tasks/GooglePlayReleaseV4/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -43,6 +43,10 @@
   "loc.input.help.shouldUploadMappingFile": "Select this option to attach your proguard mapping.txt file to your aab/apk.",
   "loc.input.label.mappingFilePath": "Deobfuscation path",
   "loc.input.help.mappingFilePath": "The path to the proguard mapping.txt file to upload. Glob patterns are supported.",
+  "loc.input.label.shouldUploadNativeMappingFile": "Upload native debug symbols",
+  "loc.input.help.shouldUploadNativeMappingFile": "Select this option to attach your native mapping file to your aab/apk.",
+  "loc.input.label.nativeMappingFilePath": "Native debug symbols file path",
```

Comment:
I think the word "mapping" can confuse someone here.
Can we consider renaming these inputs and variables like "nativeDebugSymbolsFile..."?

Reply:
Fixed, thanks